src/components/CloseModal.js

```diff
@@ -53,8 +53,8 @@ CloseModal.propTypes = {
 };
 
 CloseModal.defaultProps = {
-  title: <FormattedMessage id="wizard.closeTitle" defaultMessage="Exit source creation?" />,
-  exitTitle: <FormattedMessage id="wizard.exitText" defaultMessage="Exit" />,
+  title: <FormattedMessage id="wizard.closeTitle" defaultMessage="Cancel creating the integration?" />,
```

Comment:
@radekkaluzik maybe somebody from UX could review these bigger rewordings where are doing more than just switching Sources/Integrations. Maybe this is already present somewhere in the JIRA, but I haven't found it

Reply:
This change was done with the approval of the UX team.